### PR TITLE
BLOCKS-77 - Render formulas like in Catroid

### PR DESCRIPTION
**Pull request by whati001**
Render formulas like in Catroid. Updated catroid string files. Use 'en_AU' language as fallback for absence values in other langs. Changed default values of bricks.
Added junit test for each type formula type (list, variable, string, operator).
https://jira.catrob.at/projects/BLOCKS/issues/BLOCKS-77

### Your checklist for this pull request
- [x] Include the name of the Jira ticket in the PR’s title
- [x] Include a summary of the changes plus the relevant context
- [x] Choose the proper base branch (*develop*)
- [x] Confirm that the changes follow the project’s coding guidelines
- [x] Verify that the changes generate no compiler or linter warnings
- [x] Perform a self-review of the changes
- [x] Verify to commit no other files than the intentionally changed ones
- [x] Include reasonable and readable tests verifying the added or changed behavior
- [x] Confirm that new and existing unit tests pass locally
- [x] Check that the commits’ message style matches the [project’s guideline](https://github.com/Catrobat/Catroid/wiki/Commit-Message-Guidelines)
- [x] Make sure you use BLOCKS instead of CATROID in your commit message
- [x] Stick to the project’s gitflow workflow
- [x] Verify that your changes do not have any conflicts with the base branch
- [x] After the PR, verify that all CI checks have passed (Actions)
- [x] Post a message in the *#catblocks* [Slack channel](https://catrobat.slack.com) and ask for a code reviewer

### CODE REVIEW

File: i18n/create_json.js [hunk 1]
```diff
@@ -20,15 +22,14 @@ const { locales } = require('./lang_codes_mapping');
 // please define here the configuration if needed
 const MAPPING_FILE = path.join('i18n', 'strings_to_json_mapping.json');
 const MAPPING_COMMENT = '@';
+const MAPPING = JSON.parse(fs.readFileSync(MAPPING_FILE, { encoding: 'utf-8' }));
 
 const STRINGS_DIR = path.join('i18n', 'catroid_strings');
 const STRINGS_FILE = 'strings.xml';
+const DEF_LANG = 'values-en-rAU';
+const DEF_LANG_FILE = path.join(STRINGS_DIR, DEF_LANG, STRINGS_FILE);
```
Comment (whati001) [inline i18n/create_json.js:30]: default language

File: i18n/create_msg.js [hunk 2]
```diff
@@ -35,7 +35,7 @@ const CATBLOCK_MSGS = `
 Blockly.CatblocksMsgs = {};
 Blockly.CatblocksMsgs.locales = {};
 
-Blockly.CatblocksMsgs.currentLocale_ = 'en_GB';
+Blockly.CatblocksMsgs.currentLocale_ = 'en_AU';
```
Comment (whati001) [inline i18n/create_msg.js:38]: changed default language

File: src/js/parser/formula.js [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+    if (this.right === null) this.right = rightBlock;
+    else this.right.setRight(rightBlock);
+  }
+
+  static getAllLayouts() {
+    return {
+      'BRACKET': '(%l%r)',
+      'USER_LIST': '*%v*',
+      'STRING': '\'%v\'',
+      'USER_VARIABLE': '"%v"',
```
Comment (whati001) [inline src/js/parser/formula.js:29]: Added missing rendering layouts.

**Comment by wslany** [comment]
Great!

**Pull request merged**